packages/indexer/src/plugins/kv.test.ts

```diff
@@ -0,0 +1,74 @@
+import { type Database, open } from "sqlite";
+import sqlite3 from "sqlite3";
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import { KVStore } from "./kv";
+
+type ValueType = { data: bigint };
+
+describe("KVStore", () => {
+  let db: Database<sqlite3.Database, sqlite3.Statement>;
+  let store: KVStore;
+  const key = "test_key";
+
+  beforeAll(async () => {
+    db = await open({ driver: sqlite3.Database, filename: ":memory:" });
+    await db.exec(`
+      CREATE TABLE IF NOT EXISTS kvs (
+        from_block INTEGER NOT NULL,
+        to_block INTEGER,
+        k TEXT NOT NULL,
+        v BLOB NOT NULL,
+        PRIMARY KEY (from_block, k)
+      );
+    `);
```

Comment:
Maybe put this logic in a static `initialize(db)` method on the `KVStore`? That way we can share the logic between tests and hooks.

Reply:
Sorry I should have caught it in the previous review round!